File: lua/refactoring/tests/refactor/106/csharp/with-comments/extract.expected.cs
namespace Test
{
    class Extract
    {
        public static void foo(INSERT_PARAM_TYPE a, INSERT_PARAM_TYPE test, INSERT_PARAM_TYPE test_other)
        {
            for (int x = 0; x < test_other + test; x++) {
              Console.WriteLine(x + " " + a);
            }
        }

        /*
        * This is a comment
        * comments are fun
        * wow!
        */
        public static void simpleFunction(int a)
        {
            int test = 1, test_other = 11;
            foo(a, test, test_other);
        }
    }
}

File: lua/refactoring/tests/refactor/106/rb/with-comments/extract.expected.rb
# This is a comment
# Ruby still rocks

def foo_bar(a, test, test_other)
  [test, test_other].each do |v|
    puts "#{a} #{v}"
  end
end

# It's not dead
def simple_function(a)
  test = 1
  test_other = 11

  foo_bar(a, test, test_other)
end
